frontend/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Powerscore</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/2.0.0/modern-normalize.min.css"
+      integrity="sha512-4xo8blKMVCiXpTaLzQSLSw3KFOVPWhm/TRtuPVc4WG6kUgjH6J03IBuG7JZPkcWMxJ5huwaBpOpnwYElP/m6wg=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Teko:wght@300..700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <div class="powerscore">
+      <div class="primary-info">
+        <div class="team-score">
+          <div class="team-name">Lakeville</div>
+          <div class="score">7</div>
+        </div>
+        <div class="time">
+          <div class="timer">5.04</div>
+          <div class="period">
+            <div class="period-text">Period</div>
+            <div class="period-number">3</div>
+          </div>
+        </div>
+        <div class="team-score">
+          <div class="team-name">Eagan</div>
+          <div class="score">2</div>
+        </div>
+      </div>
+      <div class="secondary-info">
+        <div class="penalties">
+          <table>
+            <tr>
+              <th>PLYR</th>
+              <th>Penalty</th>
+            </tr>
+            <tr>
+              <td class="penalty-player">20</td>
+              <td class="penalty-time">1:42</td>
+            </tr>
+            <tr>
+              <td class="penalty-player">7</td>
+              <td class="penalty-time">1:42</td>
+            </tr>
+          </table>
+        </div>
+        <div class="shots-on-goal">
+          <div class="shots">30</div>
+          <div class="shots">24</div>
+          <div class="shots-on-goal-text">Shots On Goal</div>
+        </div>
+        <div class="penalties">
+          <table>
+            <tr>
+              <th>PLYR</th>
+              <th>Penalty</th>
+            </tr>
+            <tr>
+              <td class="penalty-player">11</td>
+              <td class="penalty-time">1:47</td>
+            </tr>
+            <tr>
+              <td class="penalty-player">62</td>
+              <td class="penalty-time">:55</td>
+            </tr>
+          </table>
+        </div>
+      </div>
+      <div class="advertisement">
```

Comment:
Let's remove the advertisement. We don't need that. That was just that image I grabbed.

Reply:
Oh good. I wondered about that.